core/src/main/java/org/radargun/config/Evaluator.java

```diff
@@ -363,6 +364,34 @@ private static Value abs(Value value) {
       }
    }
 
+   private static Value listGet(Value first, Value second) {
+      if (!second.type.canBeLong()) {
```

Comment:
If the second argument is a list, you can treat this as a multi select, e.g. (1, 2, 3, 4).get(1, 3) -> (2, 4)